### ISSUE TITLE
Play-with-docker text-file getting "snipped"

### ISSUE THREAD

**Issue by GDLynx** [play-with-docker/play-with-docker#298]
Was not sure what to call this, but basically when I run "docker-compose up", *something* decides to remove text from my file, I have tried re-adding & pasting the missing text and the same thing happens, at first I thought this was just me, but it has happened on two separate instances. 

Browser: Opera v55.0.2994.61 

Operating Sysem: Windows 10 Home v1803 
OS Build: 17134.285 

**Example:** 
```
FROM node:10

WORKDIR /usr/src/app

COPY package*.json ./

RUN npm install

COPY . .

EXPOSE 3000

CMD ["npm", "start"]
``` 
After "docker-compose up" becomes: 
```
FROM node:10

WORKDIR /usr/src/app

COPY package*.json ./

RUN npm install

COPY . .

EXPOSE 3000

CMD ["npm", "start
``` 

In the first session it happened on the "docker-compose.yml" file, but now it is happening to my Dockerfile and I am unable to determine if my Docker files are actually working correctly because I was unable to install Docker onto my local machine (hence why I am using Play-with-docker). But right now my Docker container is not running correctly though I cannot tell if it is because of the bug I am currently reporting, or if it is due to my Docker files. I am new to Docker. 

I cannot tell if it "snips" my file before or after running the container.  

**Additional Information:** 
* I was using the root directory to run my Docker instances. 
* A Node, Express, MySQL app  (using the Node and Mysql images from Dockerhub) 
* I am able to run a (git) cloned repository using "docker-compose up" without issue (using Mongo image instead) 

Note: that while I am able to run the cloned repository fine, it "snips" the Dockerfile (at least in this case it did), in the same way that it done to mine. But **given that the repository runs fine, this implies it does that "snipping" after the parsing of the files,** but I would have to re-add the missing text to run it again.

**Comment by marcosnils** [comment]
This is really strange. Haven't seen this before. Which OS are you using?

**Comment by GDLynx** [comment]
Operating Sysem: Windows 10 Home v1803 
OS Build: 17134.285

**Comment by marcosnils** [comment]
Maybe it's something related to Windows Home. Haven't tried to spin the environment in that OS :(

**Comment by GDLynx** [comment]
Maybe, I'll try on different browsers and different operating systems later tonight, but I am struggling to see how the operating system would interfere with a web-app.

**Comment by marcosnils** [comment]
I don't think the browser might be the reason. Nobody has tested this project with Docker for windows so I believe that might be the issue.

**Comment by swcurran** [comment]
Was this on labs.play-with-docker?  I've been experiencing the same thing intermittently. Running on Chrome on Windows 10 Pro. It seems to happen on the fly as I run docker-compose.

**Comment by marcosnils** [comment]
@swcurran AFAIK @GDLynx is referring to setting up PWD locally. Not using the hosted version

**Comment by GDLynx** [comment]
I was referring to the Play-with-Docker webapp via the website (so not locally).

**Comment by marcosnils** [comment]
@GDLynx got it . I thought it was something else. I'll try to reproduce it later

**Comment by swcurran** [comment]
I've had three intermittent examples of this that magically resolved a little later.  Best memory of them:

- ran `docker-compose build` on a repo and got a python error back.  I could run `docker build -t xxx <dockerfile>' directly and it worked, and `docker-compose up` worked.  Repeatable for awhile and then later it all worked properly.
- ran another docker-compose build at another time and got an error that I googled to find that docker-compose was saying "hey, your yml file is malformed`. Error indicated it was on line 381 (as I recall of the file) which was way bigger than the file.   Again, resolved sometime later.
- last was similiar, but less recollection of the error.  Tried several instance and several sessions over a period of 10 minutes and it was repeatable.  Tried a local build/run and it worked.  An hour or two later on PWD, it worked fine.

So appears to be something on PWD that is intermittent, that corrupts file and so manifests itself in several ways and later goes away.  Perhaps disk space?

BTW - just found PWD the other day.  Amazing!!!!  Awesome work.  We're thinking of self-hosting an instance to enable this for casual devs and for those limited by enterprise restrictions.  Just wow!!!
